src/app/layout/footer/footer.component.ts

```diff
@@ -14,8 +14,10 @@ export class FooterComponent implements OnInit {
     new SocialLink('Twiter', 'https://twitter.com/'),
     new SocialLink('Instagram', 'https://www.instagram.com/')
   ];
+  description: string;
 
   ngOnInit(): void {
     this.currentYear = new Date().getFullYear();
+    this.description = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
```

Comment:
Hi @kflorest, this will break the lint for the following reasons:
```
footer.component.ts[21, 1]: Exceeds maximum line length of 140
footer.component.ts[21, 24]: " should be '
footer.component.ts[21, 149]: Missing semicolon
```
I will upload the fix directly to your branch and merge it, since it isn't a big change. Take this into account for other implementations :)

Reply:
ok, thank you!